hedera-node/hedera-app/src/test/java/com/hedera/node/app/platform/event/EventMigrationTest.java

```diff
@@ -36,10 +36,12 @@
 import java.util.stream.Stream;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
 
+@Disabled("Temporarily disabling until platform refactoring is complete")
```

Comment:
Can you please create an issue to enable this? Otherwise, we may forget.

Reply:
I was supposed to create the ticket, here it is https://github.com/hashgraph/hedera-services/issues/16498